src/App.js

```diff
@@ -0,0 +1,16 @@
+import React, { Route, Routes } from 'react-router-dom';
+import Home from './components/home';
+import Details from './components/details';
+
+function App() {
+  return (
+    <div className="App">
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/details" element={<Details />} />
```

Comment:
- [ ] According to the requirements:

![image](https://github.com/Zuwairanajma/react-metric-app-capstone/assets/77069011/f4f10660-4aa6-40ac-b714-2ac7a5b4d403)

Please consider having unique routes for your details views, this means that each company should have its own unique path. React router lets you do this easily with URL params:


```js
<Route path="/details/:id" element={<Details />} />
```

Now, you just have to redirect the user to the correct path when any company card is clicked:


```js
// home.jsx
...
<NavLink to={`/details/${stock.id}`} onClick={() => dispatch(select(stock.id))} key={stock.id} className="companyListItem">
  { /* Code inside the link */ }
</NavLink>
...
```

Please consider implementing this change 🤓 🔧